app/views/vm_common/_evacuate.html.haml

```diff
@@ -0,0 +1,54 @@
+%form#form_div{:name => "angularForm", 'ng-controller' => "vmCloudEvacuateFormController"}
+  = render :partial => "layouts/flash_msg"
+  %h3
+    = _('Evacuate Host')
+  .form-horizontal
+    .form-group
+      %label.col-md-2.control-label
+        = _('Auto-select Host?')
+      .col-md-8
+        %input{:type         => "checkbox",
+               :name         => "auto_select_host",
+               'ng-model'    => "vmCloudModel.auto_select_host",
+               :miqrequired  => true,
+               :checkchange  => true,
+               'ng-disabled' => "hosts.length == 0"}
+
+        %input{:type         => "hidden",
+               :name         => "auto_select_host",
+               :value        => "on",
+               'ng-if'       => "hosts.length == 0"}
+    .form-group
+      %label.col-md-2.control-label
+        = _('On Shared Storage')
+      .col-md-8
+        %input{:type         => "checkbox",
+               :name         => "on_shared_storage",
+               'ng-model'    => "vmCloudModel.on_shared_storage"}
+    .form-group{'ng-hide' => 'vmCloudModel.on_shared_storage'}
+      %label.col-md-2.control-label
+        = _('Admin Password')
+      .col-md-8
+        %input.form-control{:type         => "password",
+                            :name         => "admin_password",
+                            'ng-model'    => "vmCloudModel.admin_password"}
+  #evacuate-select-destination-host{'ng-hide' => '!vmCloudModel.auto_select_host'}
```

Comment:
This should be reversed. Hide when auto_select_host is chosen, not when it's not chosen.